src/domain/dto/readInstalledServicesItems.go

```diff
@@ -0,0 +1,18 @@
+package dto
+
+import (
+	"github.com/goinfinite/os/src/domain/valueObject"
+)
+
+type ReadInstalledServicesItemsRequest struct {
+	Pagination           Pagination                 `json:"pagination"`
```

Comment:
I won't mention what I mentioned above on every Request/Response DTO but you must follow suit on all of them.